src/views/Conference.vue

```diff
@@ -1,36 +1,59 @@
 <template>
   <Page class="league-page-container">
+    <Loading v-show="loading" />
     <div
+      v-show="!loading"
       ref="jitsi"
       class="jitsi-container"
     />
   </Page>
 </template>
 <script>
+import BrowserWindowMessageConnection from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/connection/browser-window-message';
 import JitsiMeetExternalAPI from 'jitsi-iframe-api';
 import Page from '../components/layout/Page.vue';
 import { IS_MOBILE_DEVICE } from '../utils/util';
+import Loading from '../components/Loading.vue';
 
 export default {
   name: 'Conference',
   components: {
     Page,
+    Loading,
   },
   props: {
-    room: { type: String, required: true },
+    room: { type: String, default: '' },
   },
-  mounted() {
+  data: () => ({
+    connection: null,
+    loading: true,
+  }),
+  async mounted() {
+    this.connection = await BrowserWindowMessageConnection({
+      origin: `https://${process.env.VUE_APP_JITSI_URL}`,
+    });
+
+    this.connection.connect(({ room }) => {
+      this.$router.push({ name: 'conference', params: { room } });
+    });
+
     // eslint-disable-next-line no-new
-    new JitsiMeetExternalAPI('test.league.aeternity.org', {
+    new JitsiMeetExternalAPI(process.env.VUE_APP_JITSI_URL, {
       parentNode: this.$refs.jitsi,
       width: '100%',
       height: '100%',
-      roomName: this.room,
+      roomName: this.room || '',
```

Comment:
is this necessary as far as you already enabled `default: ''` above?

Reply:
Yes, it's the main idea for that realisation.

Reply:
Thanks